bsyncviewer/templates/case_study.html

```diff
@@ -70,9 +70,9 @@ <h5><a href="https://better-lbnl-development.herokuapp.com/docs/api/">Web API</a
                 <div class="hero-item col-md-4 text-center">
                     <span class="icon-laptop"></span>
                     <div class="hero-item-content">
-                        <h5><a href="https://demo.seed-platform.org/">The SEED platform</a></h5>
+                        <h5><a href="https://buildingdata.energy.gov/#/seed">The SEED platform</a></h5>
                         <p class="mb-2">Create and account with the SEED platform and run analysis on properties</p>
```

Comment:
I think this sentence needs to be reworked...there's a typo but more importantly, I don't think you can just create an account anymore.  Maybe something like "Find out how you can use the SEED platform to run analyses on properties"?